cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,97 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPage from '../support/pages/signUp.pageObject';
+import ArticlePage from '../support/pages/article.pageObject';
+import faker from 'faker';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('User', () => {
-  before(() => {
+  let user;
+  let name;
+  let email;
+  let password;
+  let title;
+  let description;
+  let content;
+  let tags;
+  const signUpPage = new SignUpPage();
+
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      name = faker.name.firstName();
+      email = faker.internet.email();
+      password = faker.internet.password();
+
+      cy.register(user.email, user.username, user.password);
```

Comment:
The register function is called before the visit to the registration page. The order should be reversed: first visit the page, then call the registration function.

Reply:
done

Reply:
done

Reply:
done

Reply:
done